scanner/src/main/kotlin/provenance/PackageProvenanceResolver.kt

```diff
@@ -47,11 +47,12 @@ import org.ossreviewtoolkit.utils.ort.showStackTrace
  */
 interface PackageProvenanceResolver {
     /**
-     * Resolve the [KnownProvenance] of [pkg] based on the provided [defaultSourceCodeOriginsPriority].
+     * Resolve the [Provenance] of [pkg] based on [pkg.sourceCodeOrigins] if specified, or else on
+     * [defaultSourceCodeOrigins], in order of priority.
```

Comment:
IMO the way it's currently written, it's unclear what "in order of priority" (it could also refer to `Package.sourceCodeOrigins` taking precedence over `defaultSourceCodeOrigins`). I propose to start a new sentence saying e.g. "Each source code origins are listed in order of priority."

Reply:
Similar below n the implementation.